common/models/user.js

```diff
@@ -807,6 +807,31 @@ module.exports = function(User) {
       UserModel.validatesUniquenessOf('username', {message: 'User already exists'});
     }
 
+    UserModel.once('attached', function() {
+      if (UserModel.app.get('logoutSessionsOnSensitiveChanges') !== undefined)
+        return;
+
+      g.warn([
+        '',
+        'The user model %j is attached to an application that does not specify',
+        'whether other sessions should be invalidated when a password or',
+        'an email was changed. Invalidating sessions is important for security',
+        'reasons, to allow users to recover from the situation where',
+        'an attacker got access to their account.',
+        '',
+        'We are recommending to turn this feature on by setting',
+        '"{{logoutSessionsOnSensitiveChanges}}" to {{true}} in',
+        '{{server/config.json}}, unless you have implemented your own solution',
```

Comment:
take out the `,` after `}}` and using `(unless you've implemented your own solution for token invalidation)`.